src/components/SpeedDial/SpeedDial.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import WhatsAppIcon from "@mui/icons-material/WhatsApp";
+import SpeedDial from "@mui/material/SpeedDial";
+import SpeedDialAction from "@mui/material/SpeedDialAction";
+import Avatar from "@mui/material/Avatar";
+
+const pharmacist = (name, contact) => {
+  const whatsAppTextBody = `Hi ${name}, I need your advice with dosing.`;
+  const contactNum = contact;
+  const whatsAppLink =
+    "https://api.whatsapp.com/send/?phone=" +
+    contactNum +
+    "&text=" +
+    whatsAppTextBody;
+  return whatsAppLink;
```

Comment:
```suggestion
const pharmacist = (name, contact) => {
     return `https://api.whatsapp.com/send/?phone=${contact}&text={`Hi ${name}, I need your advice with dosing.`}`
}
```

Why not just make it a single string?